### ISSUE TITLE
Any way to change keybinds?

### ISSUE THREAD

**Issue by Spanglish1337** [ollm/OpenComic#121]
Is there a way to have the pages turn with A and D? Or something like that?

**Comment by aquiab** [comment]
open the devtools with ctrl+shift+i and paste this:

```
window.addEventListener("keydown", (event) => {
  if (event.keyCode === 65) {
    reading.goPrevious()
  } else if (event.keyCode === 68) {
    reading.goNext()
  }
});
```

downside is you have to do it everytime you want to use the program, but i found it very convenient

**Comment by ollm** [comment]
Closed as completed in 0a4a4597883c7c6c837acdd61e4d80dc8c0a0ec1 and available in [v1.0.0-beta.2](https://github.com/ollm/OpenComic/releases/tag/v1.0.0-beta.2)

If you are missing any default shortcuts or any configurable action, open a new issue and I will try to add it for the next version.
